a10_octavia/controller/worker/tasks/a10_network_tasks.py

```diff
@@ -971,6 +976,17 @@ def execute(self, member, nat_flavor=None, nat_pool=None):
         if nat_pool.member_ref_count == 1:
             try:
                 self.network_driver.delete_port(nat_pool.port_id)
+                if not CONF.vthunder.slb_no_snat_support:
+                    addr_list = []
+                    start = (struct.unpack(">L", socket.inet_aton(nat_flavor['start_address'])))[0]
```

Comment:
We can create a utility function that fetches address_list from start and end addresses.

Reply:
If create new API for this please also use the API in ReserveSubnetAddressForMember()
Thanks

Reply:
Done